libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -421,6 +421,9 @@ class YulUtilFunctions
 	);
 
 private:
+	/// Special case of conversion functions - handles all array conversions.
+	std::string arrayConversionFunction(ArrayType const& _from, ArrayType const& _to);
```

Comment:
Please keep the same order of functions in the header and in the cpp file.

Reply:
Hm, `arrayConversionFunction` is before `conversionFunctionSpecial` in cpp file as well. Or there is something else I don't see here?

Reply:
Yes, there are already some irregularities. Just take care to keep it next to some function that is also next to it in the header.